docs/packaging-a-RNComponent-ios.md

```diff
@@ -0,0 +1,65 @@
+---
+id: packaging-a-RNComponent-ios
+title: Packaging a react-native component
+layout: docs
+category: Guides (iOS)
+permalink: docs/packaging-a-RNComponent-ios.html
+next: native-modules-android
+---
+
+You can also inject Native modules into react-native by creating a project in Objective-C and publish it as package in npm so that anyone can use it.
+
+*Please proceed with the following steps.*
+### Creating the package
+
+* Create a new project, use this type of naming, React-native(RN) prefix.
```

Comment:
"Create a new XCode project as depicted in the following pictures. Ensure that the name of your project is prefixed with `RN`."